src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -390,25 +390,30 @@ interface MoreInformationTextProps {
 }
 
 export const MoreInformationText = ({ content }: MoreInformationTextProps) => {
-  const lines = content
-    .replace(/<\/p>/g, '')
-    .split('<br/>')
-    .map(line => line.split('<p>'))
-    .reduce((acc, val) => acc.concat(val), [])
-
   return (
     <React.Fragment>
-      {lines.map((line: string, key: number) => {
+      {content.map((chunk: string, i: number) => {
         return (
-          <p
-            key={key}
-            sx={{
-              fontSize: 'xs',
-              my: 'xxs',
-              mx: '0'
-            }}
-            dangerouslySetInnerHTML={{ __html: line }}
-          ></p>
+          <React.Fragment key={i}>
```

Comment:
Why don't we use `<>` or just `<div>`?

Reply:
["the shorthand version does not support the key attribute"](https://thisthat.dev/react-fragment-shorthand-vs-longhand/#:~:text=is%20the%20shorthand%20tag%20for,them%20in%20a%20new%20node.&text=The%20only%20difference%20between%20them,not%20support%20the%20key%20attribute.)

Reply:
👍